app/scripts/models/reliers/oauth.js

```diff
@@ -15,9 +15,10 @@ define([
   'lib/promise',
   'lib/oauth-errors',
   'lib/relier-keys',
-  'lib/url'
-], function (_, Relier, ResumeToken, p, OAuthErrors, RelierKeys, Url) {
-  var RELIER_FIELDS_IN_RESUME_TOKEN = ['state'];
+  'lib/url',
+  'lib/constants'
+], function (_, Relier, ResumeToken, p, OAuthErrors, RelierKeys, Url, Constants) {
+  var RELIER_FIELDS_IN_RESUME_TOKEN = ['state', 'verificationRedirect'];
```

Comment:
CRSF protection is one use case for `state`, but it's use is not limited to that. I don't have all the context here, but we should always provide the `state` back to the relier (same or different browser), and then it sort out what needs to happen.